frontend/awx/views/schedules/hooks/useSchedulesFilter.tsx

```diff
@@ -1,26 +1,25 @@
-import { useMemo } from 'react';
-import { IToolbarFilter } from '../../../../../framework';
 import {
-  useDescriptionToolbarFilter,
-  useNameToolbarFilter,
   useCreatedByToolbarFilter,
   useModifiedByToolbarFilter,
 } from '../../../common/awx-toolbar-filters';
+import { useDynamicToolbarFilters } from '../../../common/useDynamicFilters';
 
 export function useSchedulesFilter() {
-  const nameToolbarFilter = useNameToolbarFilter();
-  const descriptionToolbarFilter = useDescriptionToolbarFilter();
   const createdByToolbarFilter = useCreatedByToolbarFilter();
   const modifiedByToolbarFilter = useModifiedByToolbarFilter();
 
-  const toolbarFilters = useMemo<IToolbarFilter[]>(
-    () => [
-      nameToolbarFilter,
-      descriptionToolbarFilter,
-      createdByToolbarFilter,
-      modifiedByToolbarFilter,
-    ],
-    [nameToolbarFilter, descriptionToolbarFilter, createdByToolbarFilter, modifiedByToolbarFilter]
-  );
+  const toolbarFilters = useDynamicToolbarFilters({
+    optionsPath: 'schedules',
+    preSortedKeys: ['name', 'id', 'created_by', 'modified_by'],
```

Comment:
```suggestion
    preSortedKeys: ['name', 'id', 'created-by', 'modified-by'],
```
The two additional filters were not showing up at the top of the list- small typo